homeassistant/components/onkyo/media_player.py

```diff
@@ -73,7 +73,7 @@
 PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend(
     {
         vol.Optional(CONF_HOST): cv.string,
-        vol.Optional(CONF_NAME, default=DEFAULT_NAME): cv.string,
```

Comment:
This was never used according to docs, so we are not changing the schema, just cleaning it up.